ballerina/admin.bal

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java as java;
+
+# Ballerina Service Bus connector provides the capability to access Azure Service Bus SDK.
+# Service Bus API provides data access to highly reliable queues and publish/subscribe topics of Azure Service Bus with deep feature capabilities.
+@display {label: "Azure Service Bus Administrator", iconPath: "icon.png"}
+public isolated client class ASBAdministration {
+
+    final handle adminHandle;
+    #initialize the Azure Service Bus Admin client.
```

Comment:
Missing newline

Reply:
Sure. Addressed in [1578740](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/15787407f8433542e7707533c74bdb380eb5c4b6).